storage-access.bs

```diff
@@ -195,6 +195,7 @@ When invoked on {{Document}} |doc|, the <dfn export method for=Document><code>re
 
 1. Let |p| be [=a new promise=].
 1. [=Enqueue the following steps=] on the [=permission task source=]:
+    1. If this algorithm was invoked when |doc|'s {{Window}} object did not have [=transient activation=], [=reject=] |p|.
```

Comment:
You probably also want to return p at this point or should the subsequent steps still run?

Reply:
Isn't that the same for all reject steps here? Don't they implicitly return? I know they do in our implementation, i.e. we always return after reject/resolve. If you think that it's appropriate to add explicit returns we should do that in all places, maybe in a separate PR. I'm happy to do that.

Reply:
I didn't read in detail, but yeah, that would apply to all of them in that case. A follow-up might be more appropriate in that case. In specification-land reject means https://heycam.github.io/webidl/#reject (and is ideally linked with that) which only does rejecting.

Reply:
I'll file an issue and do a PR, thanks! Would be interesting to see how other algorithms that do a lot of if-true-reject steps handle it.